### ISSUE TITLE
current time tick?

### ISSUE THREAD

**Issue by FortunateStarstar86** [d-ailin/GDN#89]
Hello!
how can I adjust the code to make it a multi-step prediction? or is it possible?
I look forward to hearing from you. Thank you very much!

**Comment by FortunateStarstar86** [comment]
![Uploading 68803657966f40a20651b43aa79aae2.png…]()

**Comment by FortunateStarstar86** [comment]
![68803657966f40a20651b43aa79aae2](https://github.com/d-ailin/GDN/assets/69461421/14a61a9a-e040-4e98-afbb-b53d54a0dcd2)

**Comment by FortunateStarstar86** [comment]
For example, in order to predict the future 10 time steps, I adjusted the training set as shown in the figure, but the "out" output by the model from GDN. py cannot match multiple time steps and the dimensions are inconsistent. How should I modify the model or adjust parameters?（I have adjusted the slide_win and slide_stride accordingly）
![68803657966f40a20651b43aa79aae2](https://github.com/d-ailin/GDN/assets/69461421/f08c79ad-c04d-45ff-9767-026f4f55b2f0)

**Comment by d-ailin** [comment]
Hi, I think it is possible, not but sure how accurate it would be. 

One way to do it is to modify the output dimension to 10 instead of 1 (which is for predicting next 10 steps instead of next 1 step). This requires modifying the model output dimension to 10 and changing the ground truth to a vector of 10 dimensions as well, such that the MSE loss should be operated on a vector of 10 dimensions during training. 

The alternative way is still using the current architecture, but you manipulate the input iteratively by appending the predicted value at T time step, and use the new input to predict the values at T+1.
